Apps/WebClient/src/ClientApp/src/models/errors.ts

```diff
@@ -12,14 +12,31 @@ export interface CustomBannerError {
     detail?: string;
 }
 
-export interface HttpError {
-    message: string;
+export class HttpError extends Error {
+    // The HTTP status code returned by the request.
+    statusCode?: number;
+
+    constructor(message: string, statusCode?: number) {
+        super(message);
+        this.statusCode = statusCode;
+    }
+}
+
+// HG API binding interface for ErrorResult model
+export interface ResultErrorDetails {
+    // API's ResultMessage mapping property. This should be treated as private and the message should be mapped to and preferred.
+    resultMessage?: string;
```

Comment:
```suggestion
    // Message that will always be populated when ResultType is Error.
    resultMessage?: string;
```

Reply:
Left over from my previous implementation thanks for catching this.